src/main/java/com/example/backendspringboottechiteasycontrollerles10/controllers/TelevisionControllerBonus.java

```diff
@@ -0,0 +1,56 @@
+package com.example.backendspringboottechiteasycontrollerles10.controllers;
+
+import com.example.backendspringboottechiteasycontrollerles10.exceptions.TelevisionNameTooLongException;
+import com.example.backendspringboottechiteasycontrollerles10.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class TelevisionControllerBonus {
+
+    public List<String> televisionDatabase = new ArrayList<>();
+
+
+    @GetMapping("/televisions")
+    public ResponseEntity<List<String>> getAllTelevisions() {
+      return ResponseEntity.ok(televisionDatabase);
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<Object> getTelevision(@PathVariable("id") int id) {
+        if (id >= 0 && id < televisionDatabase.size()) {
+            String t = televisionDatabase.get(id);
+            return ResponseEntity.ok(t);
+        } else {
+            return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+        }
+    }
+
+    @PostMapping("/televisions")
+    // waarom krijg ik een foutmelding bij null?
+    public ResponseEntity<String> createTelevision(@RequestBody String television) {
+        if (television.length() > 1000) {
```

Comment:
Als je bedoeld waarom je een foutmelding krijgt als je null invoert in postman, dat komt omdat je hier de String.length() methode gebruikt. Als jou input null is, kun je daar niet de length() methode van gebruiken en krijg je een error.